app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgGraphBuilder.java

```diff
@@ -1410,14 +1414,17 @@ private synchronized void addBgReadingValues(final boolean simple) {
                     Log.e(TAG, "Error creating back trend: " + e.toString());
                 }
 
+                boolean lowPredicted = false;
+
                 // low estimator
                 // work backwards to see whether we think a low is estimated
                 low_occurs_at = -1;
                 try {
                     if ((predict_lows) && (prediction_enabled) && (poly != null)) {
                         final double offset = ActivityRecognizedService.raise_limit_due_to_vehicle_mode() ? unitized(ActivityRecognizedService.getVehicle_mode_adjust_mgdl()) : 0;
                         final double plow_now = JoH.ts();
-                        double plow_timestamp = plow_now + (1000 * 60 * 99); // max look-ahead
+                        int low_lookahead_mins = 99;
```

Comment:
It looks like `low_lookahead_mins` is just used once. Hence, this variable extraction is not necessary.

Reply:
I wanted to keep this value easily identifiable/potentially changeable if necessary so I moved it to a static constant for high and low instead